src/pages/Homepage/Trendingitem/Trendingitem.jsx

```diff
@@ -1,7 +1,47 @@
 import React from "react";
-
+import "./Trendingitem.css";
+import Pasta from "../../../assets/pasta.jpg";
+import Burger from "../../../assets/burger.jpg";
+import Pizza from "../../../assets/pizza.png";
+import Pancake from "../../../assets/pancake.jpg";
+import Salad from "../../../assets/salad.jpg";
+import { Card } from "../../../components/Card/Card";
 const Trendingitem = () => {
-  return <div>Trending Item</div>;
+  const trendingItemList = [
+    { name: "Pizza", price: 220, productUrl: Pizza },
+    { name: "Burger", price: 110, productUrl: Burger },
+    { name: "Pancake", price: 150, productUrl: Pancake },
+    { name: "Pasta", price: 290, productUrl: Pasta },
+    { name: "Salad", price: 100, productUrl: Salad },
+  ];
+
+  const addToCart = (item) => {
+    console.log("Add to cart", item);
+  };
+
+  const addToWishlist = (item) => {
+    console.log("Add to wishlist", item);
+  };
+
+  return (
+    <section className="homepage-trending">
+      <h2>Trending Items</h2>
+      <div className="trending-item">
+        {trendingItemList.map((item) => {
```

Comment:
It's better to use one more condition inside jsx. Like {trendingItemsList && trendingItemsList.map()} using that avoid empty array using map statement. I'm using my codebase. Correct me if I'm wrong.

Remaining looking good 💯 . Good to Merge 👍

Reply:
> It's better to use one more condition inside jsx. Like {trendingItemsList && trendingItemsList.map()} using that avoid empty array using map statement. I'm using my codebase. Correct me if I'm wrong.
> 
> UI looking good 💯 . Good to Merge 👍

Good catch @santhoshpemmaka will check do this!